src/Zicht/Bundle/PageBundle/Security/Voter/PageVoter.php

```diff
@@ -28,7 +29,7 @@ public function supportsClass($class)
     public function vote(TokenInterface $token, $object, array $attributes)
     {
         // check if class of this object is supported by this voter
-        if (!$this->supportsClass(get_class($object))) {
+        if (!is_null($object) && !$this->supportsClass(get_class($object))) {
```

Comment:
i think this should be:

```
 if (is_null($object) || (!is_object($object) && !$this->supportsClass(get_class($object)))) {
```

because the method `isPublic` also expects an object